### ISSUE TITLE
Fix inaccessible website issues (missing alt text, missing labels, etc)

### ISSUE THREAD

**Issue by travis-jeans** [contribute-design/contribute.design#87]
**Describe the bug**
I was considering sending this website to a blind developer I know when I thought I would use the Orca screenreader to quickly check if the website would be accessible to them. From this quick look I found some problems, like missing labels and incorrect text alternatives.

**To Reproduce**
Open Orca Screenreader and use Ctrl+K to go to the browser URL or search bar, then use tab until you get to the website.

**Expected behavior**
Clear and easy to understand labels on icons.

**Screenshots**
The contribute.design banner is a banner missing alt text. So you just hear "blank".

<img width="835" height="181" alt="screenshot of the developer console with an svg missing a label" src="https://github.com/user-attachments/assets/73d656f2-72f8-4e82-b905-17b791c53b45" />

The GitHub icon is read as the repo name, which is confusing. It should say GitHub.

<img width="826" height="151" alt="Screenshot of an SVG icon inside a link element to the github repository" src="https://github.com/user-attachments/assets/5318b302-6295-4f18-bd63-8099d0652b06" />

There's the same problem with the other two icons, so their accessible names are basically identical. Please read this page on how to [add text alternatives to SVG](https://www.accessibilitychecker.org/wcag-guides/ensure-elements-with-an-img-graphics-document-or-graphics-symbol-role-have-an-accessible-text/) and then please fix this.

The search bar is unlabelled so you can't understand the purpose of it when navigating to it. In fact, it has an empty aria-label. The aria-label should match the placeholder text.

<img width="724" height="66" alt="the input has a blank aria-label attribute" src="https://github.com/user-attachments/assets/d74d5c0e-9383-4089-bcec-c6e887c333fd" />

**Additional context**
I didn't go further than the button because I got the impression this would take up a lot of my time. As the website creator, I ask that you please learn how to add text alternatives to improve this website.

I noticed you mention LLM in another post, so I will just say it is better to do this sort of thing by hand since you will learn a lot more from the process of adding the attributes manually and then testing it yourself with a screenreader. It's a very good habit to get into as a web-developer.

You can read more about it here: [Non-text content (WCAG)](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content), the Techniques section especially.
